lib/collection/header-list.js

```diff
@@ -0,0 +1,57 @@
+var _ = require('../util').lodash,
+    PropertyList = require('./property-list').PropertyList,
+    Header = require('./header').Header,
+
+    HeaderList;
+
+_.inherit((
+
+  /**
+   * Contains a list of header elements
+   *
+   * @constructor
+   * @extends {PropertyList}
+   */
+  HeaderList = function (parent, headers) {
+      // this constructor is intended to inherit and as such the super constructor is required to be executed
+      HeaderList.super_.call(this, Header, parent, headers);
+  }), PropertyList);
+
+_.assign(HeaderList.prototype, /** @lends HeaderList.prototype */ {
+    /**
+     * Gets size of a list of headers excluding standard header prefix.
+     *
+     * @returns {Number}
+     */
+    contentSize: function () {
+        var raw = '';
```

Comment:
`raw` can directly be assigned with `Header.unparse(this, '\r\n')`. Also, `\r\n` can be assigned to `CRLF` and reused.